src/Component/BlogPage.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react'
+import './BlogPage.css'
+
+import BlogMiddleComp from './BlogMiddleComp'
+import { data } from '../data'
+function BlogPage() {
+
+    const [searchTerm, setSearchTerm] = useState('');
+    const handleSearch = (e) => {
+        setSearchTerm(e.target.value)
+        console.log(searchTerm)
+    }
+
+    const searchFilter = () => {
+
+        let arr = data.filter(function (d) {
+            // console.log(d.name)
+            if (d.name != undefined) {
+                return d.name.toLowerCase().includes(searchTerm.toLowerCase())
+            }
+        })
+
+        return arr;
+    }
+
+    return (
+
+        <div className='blogs'>
+            {/* Seach Bar  */}
+            <input className='blogs__search__input' type="text" placeholder="@Search" onChange={(e) => handleSearch(e)} />
```

Comment:
It is not common to put `@` before placeholder text. I have never seen this before anywhere. Doesn't make it good.

Reply:
If you are using `onChange={(e) => handleSearch(e)}` and passing `e` in the function call, you could just do `onChange={handleSearch}`